recipe/meta.yaml

```diff
@@ -5,36 +5,40 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/UV-CDAT/distarray/archive/{{ version }}.tar.gz
+  url: https://github.com/CDAT/distarray/archive/{{ version }}.tar.gz
   sha256: ca6ee7610b8e8309cec47121b2454d8d318bfe37b6db3b950c65b1316035ba6c
+  patches:
+    - 0001-add-env-flag-to-disable-mpi4py.patch
 
 build:
-  number: 1
+  number: 2
   noarch: python
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
   host:
-    - python
+    - python >=3.6
     - pip
   run:
-    - python
+    - python >=3.6
     - future
 test:
     imports:
      - distarray
 
 about:
-  home: http://uvcdat.llnl.gov/
-  license: BSD 3-Clause
+  home: http://cdat.llnl.gov/
+  license: BSD-3-Clause
   license_file: 'license.txt'
   summary: 'Big Array partitionned in sub arrays'
   description: |
       Divide work among proceses for ensemble runs, linear interpolation,
       finity differencign.  Leverage the cores on the computer and is MPI ready.
-  dev_url: https://github.com/UV-CDAT/distarray
+  dev_url: https://github.com/CDAT/distarray
 
 extra:
   recipe-maintainers:
     - dnadeau4
     - doutriaux1
+    - xylar
+    - jasonb5
```

Comment:
@jasonb5, are you willing to be a maintainer?

Reply:
@xylar I'm fine being a maintainer.